### ISSUE TITLE
Loading numpy is failing on some bots

### ISSUE THREAD

**Issue by randomascii** [catapult-project/catapult#4038]
I noticed these on a try job:

________ running 'vpython.bat src/third_party/catapult/telemetry/bin/fetch_telemetry_binary_dependencies' in 'C:\b\c\b\win'
WARNING:root:Unable to import numpy due to: DLL load failed: %1 is not a valid Win32 application.
WARNING:root:Unable to import numpy due to: DLL load failed: %1 is not a valid Win32 application.
________ running 'vpython.bat src/tools/perf/fetch_benchmark_deps.py -f' in 'C:\b\c\b\win'
WARNING:root:Unable to import numpy due to: DLL load failed: %1 is not a valid Win32 application.
WARNING:root:Unable to import numpy due to: DLL load failed: %1 is not a valid Win32 application.

https://logs.chromium.org/v/?s=chromium%2Fbb%2Ftryserver.chromium.win%2Fwin7_chromium_rel_ng%2F42639%2F%2B%2Frecipes%2Fsteps%2Fgclient_runhooks__with_patch_%2F0%2Fstdout

That seems bad. I'm not sure why we're loading numpy, or why it's failing to load. Just thought I'd point this out. It looks like it's catapult/perf related, but ???

Perhaps related to issue 2865?

**Comment by kainino0x** [comment]
+@perezju for vpython?
(+me for numpy/#2865)

**Comment by perezju** [comment]
Hmm, from that bit of the log we can see that both:

* The command is already being run by vpython
* The bot has a full chromium checkout (from `src/...`)

That means vpython should be picking up its numpy version from [src/.vpython](https://cs.chromium.org/chromium/src/.vpython?q=numpy&l=43).

@riannucci can you help triage for someone in chrome-infra to help us figure out why numpy on Windows is failing to load?

**Comment by perezju** [comment]
Actually, the problem is probably that Telemetry is trying to force itself to load numpy and cv2 from [telemetry/third_party](https://cs.chromium.org/chromium/src/third_party/catapult/telemetry/third_party/) rather than the wheels provided by infra?

(or some things get loaded from vpython wheels, while others from telemetry/third_party and we get some conflicts there)

**Comment by riannucci** [comment]
Is there a monorail bug for this?

**Comment by kainino0x** [comment]
Are these bugs (#4038, #2865) obsolete now with the work on vpython (https://crbug.com/776804)?  @riannucci

**Comment by riannucci** [comment]
Yes; vpython can be set up to provide these packages reliably without requiring a compiler toolchain on the destination system. Chromium/src has wheels for opencv and numpy: https://chromium.googlesource.com/chromium/src.git/+/master/.vpython

**Comment by riannucci** [comment]
(and those wheels are now available on all builders + triggered swarming bots via vpython)

**Comment by perezju** [comment]
So it should be possible to remove now the numpy and cv2 from catapult/third_party?

**Comment by riannucci** [comment]
Yes, I believe so, though one of the catapult owners will need to weigh in on this.
